web/rest/platform/features/my/activeCalls/getActiveCallsFilters.feature

```diff
@@ -11,6 +11,20 @@ Feature: Retrieve active calls filters
       And the JSON should be equal to:
       """
       {
-          "criteria": "trunks:b1:c2:cr2:dp4:*"
+          "criteria": "trunks:-1"
+      }
+      """
+
+  Scenario: Retrieve platform active calls filter json
+    Given I add Authorization header
+     When I add "Accept" header equal to "application/json"
+      And I send a "GET" request to "/my/active_calls/realtime_filter?b=1&c=2&dp=4"
+     Then the response status code should be 200
+      And the response should be in JSON
+      And the header "Content-Type" should be equal to "application/json"
+      And the JSON should be equal to:
+      """
+      {
+          "criteria": "trunks:b1:c2:dp4:*"
       }
       """
```

Comment:
Can you also add tests for filters for a given Carrier?
Can you also add tests for DdiProvider&Direction and Carrier&Direction ? P.e. DdiProvider and direction = inbound and DdiProvider and direction = outbound.

This tests will also be interesting for the other APIs